mods/ra/rules/vehicles.yaml

```diff
@@ -480,7 +480,6 @@ MNLY:
 	UpdatesPlayerStatistics:
 		AddToArmyValue: true
 	Selectable:
```

Comment:
Can remove this line as `Selectable:` is only mentioned for `Priority:` here, and we delete that.